### ISSUE TITLE
Error executing query failed to parse query: unknown function "select"

### ISSUE THREAD

**Issue by opsnull** [influxdata/ifql#156]
```
$ tail /etc/influxdb/influxdb.conf

[continuous_queries]
  log-enabled = true
  enabled = true
  run-interval = "1s"

[ifql]
  enabled = true
  log-enabled = true
  bind-address = "10.160.109.152:8082"

$ ./ifqld --verbose --host 10.160.109.152:8082 -b 10.160.109.152:8093
2017/11/23 16:51:29 Starting version 0.0.2 on 10.160.109.152:8093
2017/11/23 16:51:31 select(db:"telegraf")
2017/11/23 16:51:31 Error: failed to parse query: unknown function "select"

$ curl -XPOST --data-urlencode 'q=select(db:"telegraf")' 10.160.109.152:8093/query
Error executing query failed to parse query: unknown function "select"
```

**Comment by gianarb** [comment]
Hello @opsnull I am sorry for this misunderstanding but we are actively developing ifql and we didn't update the README file with the last news.

As you can see here https://github.com/influxdata/ifql/pull/150 we replaced `select` with `from`.

I updated the readme here https://github.com/influxdata/ifql/pull/158

```
curl -XPOST --data-urlencode 'q=from(db:"telegraf")' 10.160.109.152:8093/query
```
Just ramamber that a `range` is required.

**Comment by nathanielc** [comment]
Fixed in #158
